src/scripts/Test-Toolkit.ps1

```diff
@@ -0,0 +1,66 @@
+param (
+    [string]$ResourceGroup,
+    [string][ValidateSet('Deploy', 'Test', 'Clean', 'Full')]$Mode = 'Full',
+    [string[]]$ExportScopes = @("providers/Microsoft.Billing/billingAccounts/8611537",
```

Comment:
Scopes should start with a `/`. Can you include that to make sure it's handled correctly? We can also do one with and one without to cover both scenarios, if you want. Add a comment to explain why we have it.